Pipfile.lock

```diff
@@ -731,10 +772,10 @@
         },
         "thoth-glyph": {
             "hashes": [
-                "sha256:405faa22285bd9e5b1cf0ad561ab618fcc3948759be931f1199236034f94fbc4"
+                "sha256:7c4560f7d327c5de74f79e7c247b7710c9438f3cc0e398337a6de33ee8698f25"
             ],
             "index": "pypi",
-            "version": "==0.0.0"
+            "version": "==0.1.0"
```

Comment:
Let's merge this as hotfix to address issue https://github.com/thoth-station/kebechet/issues/497